### PR TITLE
try to make a seed :)

### PR DESCRIPTION
**Pull request by Superbene**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,141 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+require 'faker'
+Penguin.destroy_all
+Ingredient.destroy_all
```
Comment (KimMai212) [inline db/seeds.rb:10]: Do we need that line?

Comment (Superbene) [reply db/seeds.rb:10]: ka. Ingredienz sollte eigentlich Information heißen

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,141 @@
...
+    name: "Zuza",
+    picture: '24.jpeg'
+  }
+]
+informations = ["strong", "tall", "nice", "shy", "drug addicted", "alcoholic", "smoker", "lazy", "loud", "politly", "gay", "pornstar"]
+informations.each { |information| Informations.create(name: information) }
+penguins.each { |penguin|
+    created = Penguin.create(penguin)
+    20.times do |n|
+      content = Faker::Butler.review
```
Comment (KimMai212) [inline db/seeds.rb:114]: What is `Faker::Butler.review`?

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,141 @@
...
+    20.times do |n|
+      academic_grade = ["Butler", "PhD", "Bachlor of Helping", "Dr. help. serve.", "Sir", "Master of Desaster"]cr
+      Grade.create!(
+      academic_grade: academic_grade)
+    end
+
+    3.times do |n|
+      desc = ["is a little bit", "thinks he/she is", "wishes to be ","could be", "want to be"].sample
+      information = Informations.all.sample
+      Dose.create!(
```
Comment (KimMai212) [inline db/seeds.rb:140]: Where does `Dose` come from?

**Pull request merged**